### PR TITLE
Style fix

### PR DESCRIPTION
**Pull request by thomasfanell**

### CODE REVIEW

File: source/Error/ConsoleHandler.ts [hunk 1]
```diff
@@ -13,4 +13,4 @@ module Magic.Error {
 			console.log(message.toString())
 		}
 	}
-}
\ No newline at end of file
+}
```
Comment (davidhesselbom) [inline source/Error/ConsoleHandler.ts:16]: Why does this type of change keep appearing in your commits? Does your editor automatically insert newlines at the end of files?

Comment (thomasfanell) [inline source/Error/ConsoleHandler.ts:16]: Annoyingly, no. I've looked around for a setting for this, but have come up empty. They will probably fix this, I hope.

Comment (thomasfanell) [inline source/Error/ConsoleHandler.ts:16]: There is a issue raised, they are looking into it, it seems.

Comment (davidhesselbom) [inline source/Error/ConsoleHandler.ts:16]: And if so, and if these files didn't already have newlines at the end, where do the files come from?

**Pull request merged**